js/app.js

```diff
@@ -39,6 +39,30 @@ let odd19 = new Odd('wine-glass', './images/wine-glass.jpeg');
 
 let odds = [odd1, odd2, odd3, odd4, odd5, odd6, odd7, odd8, odd9, odd10, odd11, odd12, odd13, odd14, odd15, odd16, odd17, odd18, odd19];
 
+// Question: Where in the code do I want to load my odds array?
+// Load from LS
+if (localStorage.getItem('savedOdds')) {
+  //   // Step 3.
+  let savedOdds = localStorage.getItem('savedOdds');
+  // Step 4.
+  odds = JSON.parse(savedOdds);
+}
+
+// // Question: Where in the code do I want to load my odds click count?
+// // Load from LS
+if (localStorage.getItem('countedClicks') <= 24) {
+  //   // Step 3.
+  let countedClicks = localStorage.getItem('countedClicks');
+  //   // Step 4.
+  clicks = JSON.parse(countedClicks);
+} else {
+  localStorage.removeItem('savedOdds');
+}
```

Comment:
With the way your code is currently set up, you're always adding to the data in local storage, and never clearing it. With that being said, it seems a little strange to care if the user has completed a 25 click session. Other than the context being strange, this is great code.